ui/app/clinical/consultation/controllers/drugOrderHistoryController.js

```diff
@@ -12,9 +12,10 @@ angular.module('bahmni.clinical')
             var prescribedDrugOrders = [];
             $scope.dispensePrivilege = Bahmni.Clinical.Constants.dispensePrivilege;
             $scope.scheduledDate = DateUtil.getDateWithoutTime(DateUtil.addDays(DateUtil.now(), 1));
-            $scope.enableIPDFeature = appService.getAppDescriptor().getConfigValue("enableIPDFeature");
+            $scope.autoSelectNotAllowed = appService.getAppDescriptor().getConfigValue("autoSelectNotAllowed");
```

Comment:
@abinaya-u I think "autoSelectNotAllowed" should come inside "printPrescriptionFeature" since this is specific to print prescription feature.